Simplenote/Classes/SPAuthHandler.swift

```diff
@@ -72,13 +72,20 @@ class SPAuthHandler {
             switch result {
             case .success:
                 onCompletion(nil)
-            case .failure(let statusCode, let error):
-                let error = SPAuthError(signupErrorCode: statusCode, response: nil, error: error)
-                onCompletion(error)
+            case .failure(let error):
+                onCompletion(self.handleRemoteError(error))
             }
         }
     }
 
+    private func handleRemoteError(_ error: RemoteError) -> SPAuthError {
```

Comment:
Note that this API doesn't really handles remote errors, so the name is a bit misleading. 

-`errorFromRemoteError()`
- `authenticationError(for remoteError: RemoteError)`

Or so?